scalafix-cli/src/main/scala/scalafix/internal/interfaces/ScalafixEvaluationImpl.scala

```diff
@@ -16,11 +16,11 @@ final case class ScalafixEvaluationImpl(
     fileEvaluations: Seq[ScalafixFileEvaluationImpl]
 ) extends ScalafixEvaluation {
 
-  override def isSuccessful: Boolean = error.isOk
-
   override def getErrors: Array[ScalafixError] = {
     ScalafixErrorImpl.fromScala(error)
   }
+  override def isSuccessful: Boolean =
+    getErrors.toList.filter(_ != ScalafixError.LinterError).isEmpty
```

Comment:
i think the behavior change of both `isSuccessful` is lacking a test case to prevent regressions

Reply:
ok, good idea